app/views/users/login.html.erb

```diff
@@ -6,6 +6,17 @@
         <%= image_tag("login_logo.png") %>
       </div><!--/#brand-->
     </div>
+    <% if flash_inline.inspect %>
+      <div class="col-sm-12">
+        <% flash_inline.each do |type, message| %>
+          <div class="alert <%= alert_class type %>">
```

Comment:
`alert_class type` or `alert_class(type)`, just for sake of consistency

Reply:
corrected.